### PR TITLE
Update the RDS root CA list

### PR DESCRIPTION
**Pull request by MarcoIeni**
Fix issue https://github.com/rust-lang/infra-team/issues/105
Similar to https://github.com/rust-lang/triagebot/pull/1772

### CODE REVIEW

File: database/src/pool/postgres.rs [hunk 1]
```diff
@@ -1349,3 +1358,15 @@ fn parse_artifact_id(ty: &str, sha: &str, date: Option<DateTime<Utc>>) -> Artifa
...
+
+#[cfg(test)]
+mod tests {
+    use super::make_certificates;
+
+    // Makes sure we successfully parse the RDS certificates and load them into native-tls compatible
+    // format.
+    #[test]
+    fn can_make_certificates() {
+        make_certificates();
```
Comment (Kobzol) [inline database/src/pool/postgres.rs:1370]: I know that you took this from triagebot, but can we also just do a small assert that the resulting certificate chain is not empty? Thanks!

Comment (MarcoIeni) [reply database/src/pool/postgres.rs]: Sure! Done 👍

**Comment by Kobzol** [comment]
Thanks! Let's try to deploy this and see if it works. Worst case we can just revert it.

**Review by Kobzol — APPROVED** [review]

**Comment by MarcoIeni** [comment]
Thanks, let me know once you confirm it's working 👍

**Comment by Kobzol** [comment]
The web interface seems to work fine, data from the DB is appearing. Let's see if collector can also store data there (will get results in ~1.5 hour).

**Comment by Kobzol** [comment]
Actually, it also stores data in the meantime, and the data has been updated at https://perf.rust-lang.org/status.html. So it works.

**Comment by MarcoIeni** [comment]
Great, thanks!

**Pull request merged**